ftdetect/minisnip.vim

```diff
@@ -0,0 +1,13 @@
+" Vim ftdetect file
+" Language:      minisnip
+" Created by:    Robert Hicks <gowther@zoho.com>
+" Last Change:   2017-08-20
+"
+" This simply sets all files that are in the defined
+" directory as 'minisnip'.
+"
+
+" get the snippets directory
+let snippet_dir = expand(g:minisnip_dir) . "*"
+
+execute 'autocmd BufRead,BufNewFile ' . snippet_dir . ' set filetype=minisnip'
```

Comment:
If  you are going to rewrite minisnips to use an actually directory path and file name extension (e.g. .minisnip) then maybe this is a moot effort? If you think that change will take you a while, I will try and find this answer out.